packages/api/src/get-build.js

```diff
@@ -0,0 +1,59 @@
+
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+// class to receive POSTS with build information
+
+class GetBuildHandler {
+  constructor (app, client) {
+    this.app = app;
+    this.client = client;
+  }
+
+  listen () {
+    // return the different environments for a build - all parameters and possible values for next query
+    this.app.get('/api/buildenv/:repoid', async (req, res, next) => {
```

Comment:
I think I said this in another PR, I like the approach you're taking of encapsulating routes inside classes 😄 

@cedpeters we're probably going to want to do the same for the auth work you're doing.